composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
Take a exemple:

`MessageInterface` from `psr/http-message:0.1` specifies method `addHeader($name, $value)`.
`MessageInterface` from `psr/http-message:0.2` specifies method `withHeader($name, $value)`.

If there's no version constraint in composer.json, it will break in runtime if you call a method from `0.1` and I'm implementing `0.2`.